File: docs/components_page/components/input_group/check_radio.R
library(dashBootstrapComponents)

input_groups <- div(
  list(
    dbcInputGroup(
      list(
        dbcInputGroupText(dbcRadioButton()),
        dbcInput()
      ),
      className = "mb-3",
    ),
    dbcInputGroup(
      list(
        dbcInputGroupText(dbcCheckbox()),
        dbcInput()
      )
    )
  )
)

File: docs/components_page/components/input_group/check_radio.jl
using DashBootstrapComponents, DashHtmlComponents

input_groups = html_div([
    dbc_inputgroup(
        [dbc_inputgrouptext(dbc_radiobutton()), dbc_input()],
        className = "mb-3",
    ),
    dbc_inputgroup([dbc_inputgrouptext(dbc_checkbox()), dbc_input()]),
]);
